features/works/chapter_edit.feature

```diff
@@ -640,3 +640,87 @@ Feature: Edit chapters
     When I follow "Edit Chapter"
      And I follow "Cancel"
     Then I should see the page title "Camp Friends - Chapter 2 - karma"
+
+  Scenario: Setting chapter position to 1 on new chapter correctly adjusts positions when posting and properly sends out subscription email
+    Given I am logged in as "karma"
+      And the user "second_user" with the email "second_user@example.com" exists
+      And all emails have been delivered
+      And I post the work "You and I" with chapter title "My First Chapter"
+      And "second_user" subscribes to work "You and I"
+    When I am logged in as the author of "You and I"
+      And I follow "My Dashboard"
+      And I follow "You and I"
```

Comment:
The tests can be slightly sped up by logging in as "karma" and directly going to view the work.
Also applicable to the middle position test.